### DISCUSSION TITLE
We need to work out conter-strategy.

### DISCUSSION THREAD

**Discussion by AdUser** [rms-support-letter/rms-support-letter.github.io#5205] [General]
With only-defence you can't win the war even in theory. These shit-eaters already making an execution lists:

https://github.com/sticks-stuff/highlight-RMS-supporters
https://github.com/aaronbassett/rms-letter-sigs

...and sending threats by email to known signers:

https://github.com/rms-support-letter/rms-support-letter.github.io/issues/3484

...and not are going to stop. So, why we should be tolerate?

At least we can report an abuse on users, involved in such activity (but not all signers of "open-letter") and repo with original letter. It's completely legal: all of these users/repo violates github community rules (defamation/harassment) and at least EU GDPR. If they don't want to talk[1] and discuss, we have no other options. After few hundreds of abuses on these repo/authors even github will have to react.

Next. We can initiate our own movement to seek resignation of known RMS "haters", such as Neil McGovern. His behaviour unethical and violates the same rules named above. I think this is incompatible with any manager position in any FOSS-related organization.

We are now fighting not only for RMS, we are fighting for right to call spade a spade.

[1] See https://github.com/rms-support-letter/rms-support-letter.github.io/issues/175

**Comment by hinell** [comment]
1. These execution lists pretty nicely characterize those who created & use them. They are pathetic miserable creatures who CAN'T tolerate others' opinions while WE - can do it easily. Freedom of Speech must prevail. Counteracting them the way they attack us would cast a shadow on our goals and movement.
2. The email profile was proven to be a fake one. It was created and immediately deleted (see [this reply][post]). No side so far has admitted they used intimidation tactic. So this doesn't warrant any counter measures unless situation repeats and there would be more evidence.

[post]: https://github.com/rms-support-letter/rms-support-letter.github.io/issues/3484#issuecomment-808923186

**Reply by AdUser** [reply]
> Counteracting them the way they attack us
1. That's not true. We are just trying to force everyone to follow the same rules. Harrasment is unacceptable? Well, so then users named above should be punished. Because it's applies to ALL, and not only "some we don't like".
> unless situation repeats 
2. Do you know about ESR expell from OSI in the same manner about a year ago? How much more cases do you need?  http://esr.ibiblio.org/?p=8609

**Reply by hinell** [reply]
Yeah, I heard of others cases.

**Comment by realGWM** [comment]
> shit-eaters

Just my 5 cents, but I think that saying things like that doesn't make our movement look good. :man_shrugging:

**Comment by ghost** [comment]
We should be tolerant because it is the right thing to do, both ethically and strategically.

Do not become what you hate.

**Comment by nukeop** [comment]
Focus on one thing at a time. Our goal is to ensure that the FSF confirms that they will not cave to external pressures and that they intend to keep Stallman on the board of directors. We can worry about other things when we achieve this goal. Don't get distracted by loudmouths threatening you on twitter, they are irrelevant.

**Reply by AdUser** [reply]
> Don't get distracted by loudmouths threatening you on twitter, they are irrelevant.

"Hit the shepherd and the flock will scatter"

We may loose this battle, but i'm want to win the whole war.

**Reply by jobbautista9** [reply]
The road to victory is a long one. Don't take the shortcut, it's a trap.

**Reply by hinell** [reply]
@jobbautista9 Agreed. Spreading our efforts would produce a detrimental effect for us.

**Reply by AdUser** [reply]
So maybe we take the first step (at least) on this road? For now, we are senselessly standing still and just watching.

**Reply by nukeop** [reply]
We're far from standing still, we're still promoting our letter in many ways, some of which could give us a huge boost.
